sdk/tests/wallet/chrysalis_migration.rs

```diff
@@ -384,3 +386,61 @@ fn copy_folder(src: impl AsRef<Path>, dest: impl AsRef<Path>) -> io::Result<()>
     }
     Ok(())
 }
+
+#[cfg(feature = "ledger_nano")]
+#[tokio::test]
+async fn migrate_chrysalis_db_ledger() -> Result<()> {
+    let logger_output_config = fern_logger::LoggerOutputConfigBuilder::new()
+        .name("chrysalis.log")
+        .target_exclusions(&["h2", "hyper", "rustls"])
+        .level_filter(log::LevelFilter::Debug);
```

Comment:
Do we want to keep it ? May be a bit spammish

Reply:
No, removed